frontend/src/utils/chainsInfo.ts

```diff
@@ -26,6 +26,8 @@ export const networks: Network[] = [
       chainName: 'Agoric',
       rest: 'https://agoric-api.polkachu.com',
       rpc: 'https://agoric-rpc.polkachu.com',
+      restURIs: ['https://agoric-api.polkachu.com'],
+      rpcURIs: ['https://agoric-rpc.polkachu.com'],
```

Comment:
It can be added slowly as to not change everything at once and potentially breaking something. These changes are independent so they won't effect previous code